packages/server/lib/hooks/connection/checkr-post-connection.ts

```diff
@@ -0,0 +1,32 @@
+import type { InternalNango as Nango } from './post-connection.js';
+import { isAxiosError } from 'axios';
+
+export default async function execute(nango: Nango) {
+    const connection = await nango.getConnection();
+    let access_token: string;
+    if ('access_token' in connection.credentials) {
+        access_token = connection.credentials.access_token;
+    } else {
+        return;
+    }
```

Comment:
you can slightly simplify like this I think: 
```
const access_token: string | undefined = connection.credentials.access_token;
if (!access_token) {
  return;
}
```

Reply:
Thanks. TS complains about this because access_token isn't always guaranteed to exist on credentials.